apps/api/src/modules/user/user.service.ts

```diff
@@ -29,4 +34,31 @@ export class UserService {
       }
     return userData
   }
+
+  async getPendingUsers(): Promise<PendingUserDto[]> {
+    const users = await this.repository.getPendingUsers({})
+    return users
+  }
+  async updateUserStatus(
+    userId: number,
+    updateUserStatusDto: UpdateUserStatusDto,
+  ): Promise<GetUserDto> {
+    //check if user exist
+    const user = await this.repository.getUserById(userId)
+    if (!user) throw new NotFoundException()
+
+    //accept user
+    if (updateUserStatusDto.status === UserStatus.ACCEPTED) {
+      const user = await this.repository.updateUserStatus(
+        userId,
+        updateUserStatusDto,
+      )
+      return user
+    }
+    //TODO : reject user
```

Comment:
sorry for not assign you this. I thought it will be different endpoint. If you are free you can take over the task. or just leave it for me.

Reply:
งั้นเดะเรา take over เลยก็ดีจะได้ทำทีเดียว555

Reply:
ลืมบอกว่า ขโมยแค่ [99] กับ [108] นะที่เป็น Endpoint เดียวกัน